haystack/retriever/dense.py

```diff
@@ -67,10 +68,12 @@ def __init__(self,
 
         #TODO Proper Download + Caching of model if not locally available
         if embedding_model == "dpr-bert-base-nq":
-            if not Path("models/dpr/checkpoint/retriever/single/nq/bert-base-encoder.cp").is_file():
-                download_dpr(resource_key="checkpoint.retriever.single.nq.bert-base-encoder", out_dir="models/dpr")
-            self.embedding_model = "models/dpr/checkpoint/retriever/single/nq/bert-base-encoder.cp"
-
+            import os
+            emb_path = "/media/vaishali/75f51685-53cb-4317-bd6e-dacf384b9259/vaishali/Documents/deepset/models/dpr/checkpoint/retriever/single/nq"
```

Comment:
Mhh, why is this in here? Cant we remove the download_dpr() and just use the huggingface modelhub strings and load it normally?

Reply:
Yeah, this will be removed in the model loading refactoring